simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -92,10 +92,24 @@ void start_exitByBlockNull() throws InterruptedException, ParseException, IOExce
         assertTrue(blockStreamSimulator.isRunning());
     }
 
+    void start_usingConfigurationConstructor()
+            throws InterruptedException, ParseException, IOException {
+        blockStreamSimulator = new BlockStreamSimulatorApp(configuration);
+        blockStreamSimulator.start();
+        assertTrue(blockStreamSimulator.isRunning());
+    }
```

Comment:
This ^^ method isn't used and it's not annotated with @Test so it's never called

Reply:
Good catch.